user_manual/groupware/sync_thunderbird.rst

```diff
@@ -22,7 +22,7 @@ Contacts
 
 If you later want to add a new address book, you can redo all of those steps and only the books not already synchronized will be suggested.
 
-.. note:: If your account uses two-factor authentication, you need a `dedicated app password <../session_management.html#managing-devices>`_ for login rather than your
+.. note:: If your account uses two-factor authentication, you need a :ref:`dedicated app password <#managing_devices>` for login rather than your
```

Comment:
I believe the syntax is incorrect with the #. See https://github.com/nextcloud/documentation/blob/585258b12d97924ff36689f2c46659319d26e7bc/user_manual/groupware/sync_osx.rst?plain=1#L29 as a reference.